src/locales/nl/pages.yaml

```
@@ -151,22 +151,79 @@ hire-ensemble:
 
-    Als donateur van Krashna heeft u vanaf een bepaald bedrag recht op vrijkaarten voor de concerten. Als u €50,- per academisch jaar doneert, ontvangt u twee vrijkaartjes. Als u mailt naar [krashna@tudelft.nl](mailto:krashna@tudelft.nl) zullen we u een machtigingsformulier toesturen. Dit kunt u ingevuld en ondertekend weer aan ons retourneren. Daarnaast is het ook mogelijk om uw donatie over te maken naar `NL55 RABO 0313 4648 47` ten name van DSMG Krashna Musika o.v.v. “Donateur van Krashna” en uw naam. Als u dan gebruik wilt maken van de vrijkaarten kunt u een email sturen.
+    ![Tiers](/images/donateNL.svg?style=centerme-90)
+
+    ## Eenmalige donatie
+    Daarnaast kunt u ook eenmalig een donatie doen door een bedrag over te maken naar ```NL55 RABO 0313 4648 47``` ten name van DSMG Krashna Musika o.v.v. “Donateur van Krashna” en uw naam.
+
+    ## Hartelijk dank voor uw steun!
 
 rent-instrument:
   title: Instrumentenverhuur
   text: |
     ### Huur een instrument!
 
-    Krashna Musika is in het bezit van diverse instrumenten voor haar eigen repetities en concerten. Deze instrumenten zijn ook te huur voor kortstondig gebruik door andere verenigingen, bedrijven of particulieren. Natuurlijk moet er wel naar de planning gekeken worden. Te huren instrumenten zijn onder andere: een contrabas, altviool, basklarinet, A-klarinet, piccolo, tamtam, pauken en divers ander slagwerk.
+    Krashna Musika is in het bezit van diverse instrumenten voor haar eigen repetities en concerten. Deze instrumenten zijn ook te huur voor kortstondig gebruik door andere verenigingen, bedrijven of particulieren. Natuurlijk moet er wel naar de planning gekeken worden.
+
+    Bent u op zoek naar een instrument om kortstondig te huren? Neem hiervoor [contact](mailto:orkestcom@krashna.nl) op met het bestuur. Vermeld hierbij aub om welk(e) instrument(en) het gaat en voor welke periode. U krijgt dan zo snel mogelijk reactie of dit mogelijk is en voor welk bedrag.
+
+  strings-title: Strijkinstrumenten
+  wood-title: Hout
+  brass-title: Koper
+  percussion-title: Slagwerk
+  other-title: Overig
+
+  strings: |
+
+    * Altviool
+    * Altviool strijkstok
+    * Contrabas
+    * Contrabas strijkstok
+
+  wood: |
+
+    * A-klarinet
+    * Es-klarinet
+    * Basklarinet
+    * Piccolo
+    * Althobo
+    * Contrafagot
 
-    Bent u op zoek naar een instrument om kortstondig te huren? Neem hiervoor [contact](/contact) op met het bestuur. Vermeld hierbij aub om welk(e) instrument(en) het gaat en voor welke periode. U krijgt dan zo snel mogelijk reactie of dit mogelijk is en voor welk bedrag.
+  brass: |
+
+    * Bastrombone
+
+  percussion: |
+
+    * Pauken
+    * Grote trom
+    * Xylofoon
+    * Klokkenspel
+    * Snare drum
+    * Tenor drum
+    * Bekkens
+    * TamTam
+    * Triangels
+    * Bongo's
+    * Klein Slagwerk
+    * Rommel Krat
```

```suggestion
    * Rommelkrat
```

Also, wat is een rommelkrat? Waar kun je dat voor gebruiken?

---

Not an instrument, but a crate/box with all sorts of little things needed for percussion